.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
```

Comment:
triggers on both push and pr - do we need both of them? Or which one we should keep?